docs/about/index.rst

```diff
@@ -8,6 +8,29 @@ About
 Mission
 -------
 
+Fairlearn is an open source toolkit that seeks to empower data scientists and
+developers to assess and improve fairness of their AI systems.
+The toolkit aspires to include:
+
+- Fairness metrics and unfairness mitigation algorithms
+- Interactive dashboard for fairness assessment
```

Comment:
Nitpick: I think this should be "An interactive dashboard"

Reply:
Should this be so specificly towards a dashboard, or should it simply have "methods for fairness assessment"

Reply:
@MBrouns assessment can be just with metrics as in the point above, or with something that goes beyond just numbers. There are quite a few people interested in working on seeing what can be done with an interactive UI from an HCI research perspective. Not sure whether that alleviates your concern...
Perhaps "dashboard" isn't the most intuitive word?

Reply:
@romanlutz good point, and I agree but I'm not sure dashboard necessarily covers that either. I always try to keep specific solutions outside of project goals, as solutions should be allowed to change much more often than goals. I'm also not exactly sure how I would phrase it

Reply:
How about: "Tools for interactive fairness assessment (e.g., dashboard, plotting, etc.)"